fusillade/api/evaluate.py

```diff
@@ -17,7 +17,9 @@ def evaluate_policy_api(token_info, body):  # TODO allow context variables to be
         try:
             authz_params = get_resource_authz_parameters(body['principal'], body['resource'])
         except AuthorizationException:
-            response = {'result': False, 'reason': "The user is disabled."}
+            response = {'result': False, 'reason': "UserDisabled"}
```

Comment:
I would expect this error message to be "User is not authorized" or similar - does "disabled" better describe this outcome?

Reply:
Not authorized is implied by the result false. The reason gives more details to the server side applications as to why they are not authorized.